src/main/java/seedu/address/model/review/Review.java

```diff
@@ -193,6 +193,10 @@ public void markCurrCardAsWrong() {
         updateReviewStatsList();
     }
 
+    public void setCard(Tag tag) {
+        currCard = new Card(currCard.getQuestion(), currCard.getAnswer(), tag, currCard.getDeck());
```

Comment:
I may be wrong but I think this method is unnecessary. The field currCard only stores a reference of the card. If you assign the currCard to a different card, the original card whose reference was stored in currCard will not be modified. 

And I dont think there is a reason to change the reference inside currCard, since it will not be reflected in the UI anyway. Correct me if im wrong tho sirr

Reply:
I explain this below tgt with the other command.